src/components/program/ProgramFAQ.jsx

```diff
@@ -0,0 +1,36 @@
+import { useContext } from 'react';
+import { Collapsible } from '@edx/paragon';
+import { faChevronUp, faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ProgramContext } from './ProgramContextProvider';
+import './styles/index.scss';
+
+function ProgramFAQ() {
+  const { program: { faq } } = useContext(ProgramContext);
+  if (faq && faq.length > 0) {
```

Comment:
Same feedback as above, move the statement `faq?.length > 0` inside `return` block